fuel-vm/src/tests/metadata.rs

```diff
@@ -551,19 +573,6 @@ fn get_transaction_fields() {
     );
     let script_data: Vec<u8> = cases.iter().flat_map(|c| c.iter()).copied().collect();
 
-    // Maybe use predicates to check create context?
-    // TODO GTFArgs::CreateBytecodeLength
-    // TODO GTFArgs::CreateBytecodeWitnessIndex
-    // TODO GTFArgs::CreateStorageSlotsCount
-    // TODO GTFArgs::CreateSalt
-    // TODO GTFArgs::CreateStorageSlotAtIndex
-    // TODO GTFArgs::OutputContractCreatedContractId
-    // TODO GTFArgs::OutputContractCreatedStateRoot
-
-    // blocked by https://github.com/FuelLabs/fuel-vm/issues/59
-    // TODO GTFArgs::InputCoinTxPointer
-    // TODO GTFArgs::InputContractTxPointer
```

Comment:
Why do we remove these todos?=D

Reply:
Because I added all the tests to tests them all in the PR héhé

Reply:
I don't see where we test `CreateBytecodeLength`, `CreateBytecodeWitnessIndex`, `CreateSalt`, `CreateStorageSlotAtIndex`, `OutputContractCreatedContractId`, `OutputContractCreatedStateRoot`, `InputCoinTxPointer`, `InputContractTxPointer` GTF opcodes. Could you point to corresponding tests please?

Reply:
It starts here : https://github.com/FuelLabs/fuel-vm/blob/067d836a337472be2d8b9b8daba9d99c9aa8e784/fuel-vm/src/tests/metadata.rs#L1025 and there is a test for each gtf.

Side node : InputContractTxPointer and InputCoinTxPointer do not exist anymore.

Reply:
My bad, search in browser was not working=( Thank you!